src/main/kotlin/interactor/GetTop10CountriesEnforceHighTaxesOnCarbonatedDrinksInteractor.kt

```
@@ -0,0 +1,32 @@
+package interactor
+
+import model.CityEntity
+
+class GetTop10CountriesEnforceHighTaxesOnCarbonatedDrinksInteractor (
+    private val dataSource: CostOfLivingDataSource,
+){
+
+    fun execute(limit:Int):List<Pair<String , Float?>>?{
+        val list = dataSource.getAllCitiesData()
+        if (list.isEmpty()) return null
```

here u should throw exception rather than returning null